specs/darwin/ssdeeep.table

```diff
@@ -0,0 +1,12 @@
+table_name("ssdeep")
+description("Filesystem ssdeep data.")
+schema([
+    Column("path", TEXT, "Must provide a path or directory", index=True, required=True),
+    Column("directory", TEXT, "Must provide a path or directory", required=True),
```

Comment:
Just a quick question. What the difference between `path` and `directory` columns ? And why one column with full path is not enough?